bin/snapshot-backups/stw

```diff
@@ -1,17 +1,27 @@
 #!/bin/sh
 
-SCRIPT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
-if [ -f $SCRIPT_DIR/../deploy/includes ]; then
-  . $SCRIPT_DIR/../deploy/includes
+usage="
+$0: Creates a tag for snapshot scripts and runs the selected script.
```

Comment:
Please find out what "stw" stands for and include that explanation here!

Reply:
The universe has many mysteries.